src/django_idom/config.py

```diff
@@ -10,6 +10,7 @@
 IDOM_BASE_URL = getattr(settings, "IDOM_BASE_URL", "_idom/")
 IDOM_WEBSOCKET_URL = IDOM_BASE_URL + "websocket/"
 IDOM_WEB_MODULES_URL = IDOM_BASE_URL + "web_module/"
+IDOM_WS_RECONNECT_TIMEOUT = getattr(settings, "IDOM_WS_RECONNECT_TIMEOUT", 604800)
```

Comment:
So I need to rename this parameter in the JS client. This actually defines the ceiling for the delay between reconnection attempts. The client doesn't actually have a way to stop reconnection attempts at the moment :/

Maybe `IDOM_WS_MAX_RECONNECT_DELAY` would better communicate this.

Reply:
I'll also PR IDOM core to get a true "timeout".

Reply:
I'd actually like to modify how reconnections happen so its more configurable. It should accept a callback that receives the last delay, and the total elapsed time as its arguments and returns the next delay.